packages/mux-audio/package.json

```diff
@@ -27,8 +30,12 @@
   },
   "devDependencies": {
     "@mux-elements/polyfills": "0.1.0",
+    "@open-wc/testing": "^3.0.3",
+    "@web/dev-server-esbuild": "^0.2.16",
+    "@web/test-runner": "^0.13.26",
     "esbuild": "^0.13.13",
     "npm-run-all": "^4.1.5",
+    "replace": "^1.2.1",
```

Comment:
non-blocking: We've now got `devDependencies` on `copyfiles` and `replace`. We may want to just consider adding `shx` and using it for our "shell-equivalent use cases"

Reply:
shx looks cool, I'll do on next iteration 👍